File: Iniciante/1009 - Salário com Bônus/Python/1009.py
def main():
    seller_name = input()
    fixed_salary = float(input())
    total_sales = float(input())
    COMMISSION_RATE = 0.15
    commission = total_sales * COMMISSION_RATE
    total_salary = fixed_salary + commission
    print(f"TOTAL = R$ {total_salary:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1009 - Salário com Bônus/JavaScript/1009.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const sellerName = lines.shift();
  const [fixedSalary, totalSales] = lines.map(Number);
  const COMMISSION_RATE = 0.15;
  const commission = totalSales * COMMISSION_RATE;
  const totalSalary = fixedSalary + commission;
  console.log(`TOTAL = R$ ${totalSalary.toFixed(2)}`);
}

main();
